site/layouts/_default/_markup/render-blockquote.html

```diff
@@ -0,0 +1,28 @@
+{{ $emojis := dict
+  "caution" ":exclamation:"
```

Comment:
How about these, so we dont get 3x info source?
```
  "caution" ":bangbang:"
  "important" ":information_source:"
  "note" ":page_facing_up:"
  "tip" ":bulb:"
  "warning" ":warning:"
```

Reply:
![Screenshot 2025-02-26 at 17-46-02 ZAP – ZAP Updates - January 2025](https://github.com/user-attachments/assets/8f3fb664-696f-41a4-88d5-332f92ead7ef)

Reply:
Thank you, I figured there must be something better 😁

Reply:
Much better, thanks! I'll add those in a bit.